### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by gdude2002**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Before submitting your issue, please check the following:
+
+* Ensure you are using the latest version of ProtocolSupport from Jenkins: https://build.true-games.org/job/ProtocolSupport/
+* Ensure you are using the latest version of Spigot or Glowstone as appropriate
```
Comment (MrPowerGamerBR) [inline .github/ISSUE_TEMPLATE.md]: > Ensure you are using the latest version of Spigot or Glowstone as appropriate

Shouldn't Paper be added too? Sometimes there is some Paper specific bugs

Comment (gdude2002) [inline .github/ISSUE_TEMPLATE.md]: Done

**Comment by Shevchik** [comment]
Maybe a note that if possible you should reproduce your issue with debug enabled (/ps debug)

**Comment by gdude2002** [comment]
Good idea, added

**Pull request merged**